### DISCUSSION TITLE
Nextcloud LXC Alpine

### DISCUSSION THREAD

**Discussion by sic699** [tteck/Proxmox#3079] [Help Needed]
Hi, 

I am unable to access my nextcloud server the last couple of days and i am not sure why I keep getting the following error

Error

    Application files_reminders (files_reminders) is not present or has a non-compatible version with this server. Please check the apps directory.

When I try to upgrade the nextcloud lxc alpine via console with the following command 

bash -c "$(wget -qO - https://github.com/tteck/Proxmox/raw/main/ct/alpine-nextcloud.sh)"

 (*) 1  Update Nextcloud to v29.0.0 

I get the following error  

Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
Setting log level to debug
Turned on maintenance mode
Exception: Updates between multiple major versions and downgrades are unsupported.
Update failed
Maintenance mode is kept active
Resetting log level

[ERROR] in line 85: exit code 0: while executing command su nextcloud -s /bin/sh -c 'php82 /usr/share/webapps/nextcloud/occ upgrade'

Any help would be appreciated thank you.

**Comment by tteck** [comment]
https://github.com/tteck/Proxmox/issues/3068
